### ISSUE TITLE
flux:form.container throws Exception inside fluidcontent element

### ISSUE THREAD

**Issue by tpinne** [FluidTYPO3/flux#1144]
This is a minimal example to reproduce the problem. Started with a complexer one, but this also throws the exception when trying to open the backend form.

TYPO3 7.6.7
Flux 7.4.0
Fluidcontent 4.4.1

```xml
<f:section name="Configuration">
		<flux:form id="tabContainer" options="{group: 'FCE'}">
            <flux:form.container name="settings.test" label="Test">
                <flux:field.input name="title" />
            </flux:form.container>
		</flux:form>
	</f:section>
```

throws

```
#1: PHP Warning: Invalid argument supplied for foreach() in /var/www/vendor/typo3/cms/typo3/sysext/backend/Classes/Form/Container/FlexFormElementContainer.php line 48
```

**Comment by tpinne** [comment]
I think this is the same as #992 ?

**Comment by NamelessCoder** [comment]
Should be solved by solution to #992 but I'm afraid I can't confirm.

Unfortunately some combinations of fields are no longer possible or result in incorrect behaviors in the new FormEngine. This one could be one of them (a case that's considered invalid but does not yet, or does not on your version of TYPO3, throw one of the new Exceptions to indicate invalid use). If that's the case, our official decision will be to accept that this is no longer possible (and let the TYPO3 docs explain why and what to do as alternative).
